src/analysis/retail/evoker/shared/modules/normalizers/EssenceBurstCastLinkNormalizer.ts

```diff
@@ -180,6 +183,18 @@ const EVENT_LINKS: EventLink[] = [
       return healProcDiff < castProcDiff;
     },
   },
+  {
+    linkRelation: ESSENCE_BURST_CONSUME,
+    reverseLinkRelation: ESSENCE_BURST_CONSUME,
+    linkingEventId: [SPELLS.ESSENCE_BURST_AUGMENTATION_BUFF.id, SPELLS.ESSENCE_BURST_DEV_BUFF.id],
+    linkingEventType: [EventType.RemoveBuff, EventType.RemoveBuffStack],
+    referencedEventId: [TALENTS.PYRE_TALENT.id, SPELLS.DISINTEGRATE.id, TALENTS.ERUPTION_TALENT.id],
```

Comment:
Since this is a shared module, can you also add ECHO and EMERALD_BLOSSOM. There's a similar link already in the pres module that we could probably refactor to use this for simplicity